cmd/podman/buildfarm/create.go

```diff
@@ -0,0 +1,83 @@
+package buildfarm
+
+import (
+	"fmt"
+
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/common/pkg/config"
+	"github.com/containers/podman/v4/cmd/podman/registry"
+	"github.com/containers/podman/v4/cmd/podman/utils"
+	"github.com/containers/podman/v4/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+var (
+	buildfarmCreateDescription = `Create a new farm with connections added via podman system connection add.
+
+	The "podman system connection add --farm" command can be used to add a new connection to a new or existing farm.`
+
+	createCommand = &cobra.Command{
+		Use:               "create [options] NAME [CONNECTIONS...]",
+		Args:              cobra.MinimumNArgs(1),
+		Short:             "Create a new farm",
+		Long:              buildfarmCreateDescription,
+		RunE:              create,
+		ValidArgsFunction: completion.AutocompleteNone,
+		Example: `podman buildfarm create myfarm connection1
+  podman buildfarm create myfarm`,
+	}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Command: createCommand,
+		Parent:  buildfarmCmd,
+	})
+	flags := createCommand.Flags()
+	flags.SetNormalizeFunc(utils.AliasFlags)
```

Comment:
Is this necessary? Do we already have aliases for buildfarm create?

Reply:
nope, removed